serverless/components
# Implement output type validation

## What has been implemented?

Implements the `outputTypes` YAML property which can be used to define and check component outputs.

Closes SC-94

## Steps to verify

Modify the `tests-integration-function-mock` to require some ouput types which are not present and run the `simple` integartion test or deploy the `tests/integration/simple` service.

You might also want to `deploy` and `remove` the `retail-app` to see that everything still works fine.

## Todos:
